contracts/periphery/contracts/s2s/baseapps/AppShare.sol

```diff
@@ -0,0 +1,16 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity >=0.6.0;
+
+abstract contract AppShare {
+    bytes4 internal constant DARWINIA_CHAIN_ID = 0x64617277; // darw
+    bytes4 internal constant CRAB_CHAIN_ID = 0x63726162; // crab
+    bytes4 internal constant PANGORO_CHAIN_ID = 0x70616772; // pagr
+    bytes4 internal constant PANGOLIN_CHAIN_ID = 0x7061676c; // pagl
+    bytes4 internal constant PANGOLIN_PARACHAIN_CHAIN_ID = 0x70676c70; // pglp
+    bytes4 internal constant CRAB_PARACHAIN_CHAIN_ID = 0x63726170; // crap
+
+    // Lane ids
+    bytes4 internal constant ZERO_LANE_ID = 0x00000000;
+    bytes4 internal constant ROLI_LANE_ID = 0x726f6c69; // roli
```

Comment:
Add
pangolin<>pangoro
carb<>darwinia
crab<>crab parachain

Reply:
Do the three bridges use the same 'roli' lane id?

Reply:
No，they are different